src/main/java/frc/robot/Constants.java

```diff
@@ -161,10 +161,10 @@ public static final class AutoConstants {
                                                            // but spinning fast is not particularly useful or driver
                                                            // friendly
 
-    public static final double kPXController = 2;
-    public static final double kDxController = 0.00005;
+    public static final double kPXController = 1.5;
+    public static final double kDxController = 0.000075;
```

Comment:
Were these numbers tuned to protobot?  Suggest not carrying this forward as comp is likely to be different.

Reply:
um, well, yes. on protobot. we should change these probbaly